draft-ietf-rtcweb-jsep.xml

```diff
@@ -1403,7 +1403,13 @@ candidate:1 1 UDP 1694498815 192.0.2.33 10000 typ host
           MediaStreamTrack is added to a PeerConnection, the resultant m=audio
           section will precede the m=video section. If a second MediaStream
           containing an audio MediaStreamTrack was added, it would follow
-          the m=video section.</t>
+          the m=video section. If any extra m= sections are required because
+          OfferToReceiveAudio (<xref target="sec.offertoreceiveaudio"/>
```

Comment:
I think it would make more sense to change 
   OfferToReceiveAudio were used 
to something like 
  value set for the the OfferToReceiveAudio option. 
And them a similar change for the other places in the PR